bloom-housing/bloom
# feat: upgrade to node 18 and next 12

# Pull Request Template

## Issue Overview

This PR addresses #issue


## Description

This PR upgrades node to the latest stable version of 18. In doing so it required me to upgrade next to at least 12. Since I ran into issues trying to upgrade to 13 I decide this is good enough as a first step for next migration.

## How Can This Be Tested/Reviewed?

In order for these changes to work locally you will need to update your version of node locally to 18 and then do a `yarn install`. All pages should still work and you shouldn't see any errors. 

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
